src/firefly/js/drawingLayers/ComputeWebGridData.js

```diff
@@ -73,6 +73,8 @@ import { getDrawLayerParameters} from './WebGrid.js';
 import {Regrid} from '../util/Interp/Regrid.js';
 import {getPointMaxSide} from  '../visualize/HiPSUtil.js';
 import {convert} from '../visualize/VisUtil.js';
+import {isHiPS, isHiPSAitoff} from 'firefly/visualize/WebPlot.js';
+import {getFoV} from 'firefly/visualize/PlotViewUtil.js';
```

Comment:
I think you should use relative path within firefly instead of the `firefly` alias.

Reply:
yes, that is a mistake.